coverage-tests.js

```diff
@@ -284,6 +284,41 @@ test('check window fully on android chrome emulator', {
   },
 })
 
+test('check window UFG native', {
+  variants: {
+    'on ios': {
+      env: {
+        device: 'iPhone 12',
+        app: 'https://applitools.jfrog.io/artifactory/Examples/DuckDuckGo-instrumented.app.zip',
+        injectUFGLib: true,
```

Comment:
In the interest of adding test coverage for this right now, an incremental step we can take is to just use a dedicated device which has the caps already injected. Like [this](https://github.com/applitools/eyes.sdk.javascript1/blob/master/js/packages/test-utils/src/parse-env.js#L52-L69). Thoughts @amitzur @kyryloonufriiev? Currently, NMG is not tested as part of an SDK release (either in core or by a GCT).